app/mailers/verification_code_mailer.rb

```diff
@@ -11,6 +11,18 @@ def with_code
     mail(to: params[:to], subject: @subject, from: service.noreply_email, delivery_method_options: service.delivery_method_options)
   end
 
+  def with_code_fyst
+    service = MultiTenantService.new(params[:service_type])
+    @service_name = service.service_name
+    @service_type = service.service_type
+    @locale = params[:locale]
+    @subject = I18n.t("messages.verification_code_subject_with_service_name", service_name: @service_name, locale: @locale)
+    @verification_code = params[:verification_code]
+    attachments.inline['logo.png'] = service.email_logo
+
+    mail(to: params[:to], subject: @subject, from: service.noreply_email, delivery_method_options: service.delivery_method_options)
+  end
+
```

Comment:
`with_code_fyst` looks like the exact same logic as `with_code` above. The templates appear nearly identical too. I think it would make sense to just use or modify the original template and `with_code` method. I don't think we should add a new method that does the same thing. Am I missing something that can't already be handled by the existing method and template?

Reply:
The main problem here is that the erb template is dictated by the name of this method and I thought a semi repetitive method was a lesser evil than having a conditional in the view.  If there was is way to specify which view is triggered fir the body of the email here, I could get rid of this method altogether.

Reply:
I've tried also using the render template method here but that brings about another can of worms and Tiffany has mentioned that we don't intertwine the statefile template with ctc & gyr.

@mpidcock